app/helpers/oaisys/list_identifiers_helper.rb

```diff
@@ -6,7 +6,9 @@ def resumption_token(xml_object:, complete_list_size:, cursor:, resumption_token
     if last_page
       xml_object.tag!('resumptionToken', completeListSize: complete_list_size, cursor: cursor)
     else
-      xml_object.tag!('resumptionToken', { completeListSize: complete_list_size, cursor: cursor }, resumption_token)
+      expiration_date = (Time.current + 72.hours).utc.xmlschema
```

Comment:
Might actually be worth pulling the expiration time (72 hours) into the Rails config file, on further reflection, so that we can adjust this if necessary without needing to do a full redeploy?